app/views/event_attendances/index.html.erb

```diff
@@ -1,91 +1,111 @@
 <h3 class="pre-title"><%= @event.name %></h3>
 <h1><%=h @audience.to_s %> Attendance</h1>
-	
+
 <table class="attendees">
-	<thead>
```

Comment:
@kritinvij Could you please go through these lines and change the spaces back to tabs, to keep the diff a little smaller?